components/Card.js

```diff
@@ -1,12 +1,29 @@
 import Image from "next/image";
 import styled from "styled-components";
 
+const CardImage = styled(Image)`
+  border-radius: 10px;
+  margin: 10px;
+  box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.3);
+  width: 100%;
+  min-width: 180px;
+  max-width: 180px;
```

Comment:
this also makes no sense. You can use width: 180px. What Felix ment was: You can style this CardImage to have a width: 100% (what makes it appear as wide as it can be) and also have a max-width: 180px (so it will take the maximum place it can get until it is 180px wide)

Reply:
Thanks Roland, that was a great idea. now I have a MovieList component